testgrid/config.yaml

```diff
@@ -1,5 +1,7 @@
 # Dashboards need to be specified here to be created on TestGrid
 # A prow annotation will be invalid if it references a dashboard that doesn't exist
 dashboards:
+  - name: google-gcp-guest
```

Comment:
Style is generally to keep list items indented at the same level as the key, thus:
```yaml
dashboards:
- name: foo
- name: bar
```

Reply:
Done